src/System.ComponentModel.Annotations/tests/PhoneAttributeTests.cs

```diff
@@ -23,6 +23,13 @@ protected override IEnumerable<TestCase> ValidValues()
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext 123");
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext.123");
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext. 123");
+            yield return new TestCase(new PhoneAttribute(), "1");
+            yield return new TestCase(new PhoneAttribute(), "+4+2+5+-+5+5+5+-+1+2++1+2++");
+            yield return new TestCase(new PhoneAttribute(), "425-555-1212    ");
+            yield return new TestCase(new PhoneAttribute(), " \r \n 1  \t ");
+            yield return new TestCase(new PhoneAttribute(), "1-.()");
+            yield return new TestCase(new PhoneAttribute(), "(425555-1212");
+            yield return new TestCase(new PhoneAttribute(), ")425555-1212");
```

Comment:
I wanted to see what kind of invalid phone numbers/false positives PhoneAttribute would accept!